ckanext/datajson/plugin.py

```diff
@@ -417,21 +417,32 @@ def _get_ckan_datasets(org=None, with_private=False):
         page = int(request.params.get('page', 1))
```

Comment:
Now that I'm looking at the page parameter we should be doing some validation since it's coming from users.
We can add a function to do basic validation
```
    def get_valid_page(request_page):
        page = 1
        try:
            page = int(request_page)
            if page < 1:
                page = 1
        except (ValueError, TypeError):
            pass
        return page
```

Reply:
This will be handle in another ticket